cmd/daemon/main.go

```diff
@@ -414,24 +425,46 @@ func (d *daemon) make(c *cli.Context) error {
 		}
 	}()
 
-	log.Infof("started swapd with basepath %s",
-		cfg.Basepath,
-	)
+	log.Infof("started swapd with data-dir %s", cfg.DataDir)
 	return nil
 }
 
-func newBackend(ctx context.Context, c *cli.Context, env common.Environment, cfg common.Config,
-	chainID int64, devXMRMaker bool, sm swap.Manager, net net.Host) (backend.Backend, error) {
+func errFlagsMutuallyExclusive(flag1, flag2 string) error {
+	return fmt.Errorf("flags %q and %q are mutually exclusive", flag1, flag2)
```

Comment:
I wasn't sure whether to quote the flag name or use a leading `--`. If urfave generates the error for a required flag, it is using quoting like I did above. In any case, it's in a function so we can change it in one place if people don't like it.

Reply:
seems fine to me!